index.html

```diff
@@ -6,12 +6,32 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <link rel="stylesheet" href="style.css">
-    <title>Document</title>
+    <title>Number Guesser</title>
 </head>
 
 <body>
+    <main>
+        <title>Number Guesser Game!</title>
+        <h1>Number Guesser Game!</h1>
+        <header>To play the game:</header>
+            <ol>
+                <li class="list">Consider numbers between 0 and 10</li>
+                <li class="list">Input your guess for which number is correct</li>
+                <li class="list">You can only guess 5 times, so consider the number of guesses too high or too low to help you improve your chances of winning</li>
+            </ol>
+        <input type="number" id="number-guess">
```

Comment:
You could use something like ``` <input type="number" id="number-guess" min="1" max="10"> ``` to make it so the user can't use the arrows in your input to choose a number out of range.